allauth/account/forms.py

```diff
@@ -46,7 +48,12 @@ def _make_hash_value(self, user, timestamp):
         return ret
 
 
-default_token_generator = EmailAwarePasswordResetTokenGenerator()
+if hasattr(settings, "ACCOUNT_PASSWORD_RESET_TOKEN_GENERATOR"):
```

Comment:
Can you please move this block as a @property into allauth/account/app_settings.py::PASSWORD_RESET_TOKEN_GENERATOR ?